### DISCUSSION TITLE
[DISCUSSION] Join the ASF Incubator?

### DISCUSSION THREAD

**Discussion by tisonkun** [poem-web/poem#929] [General]
Poem Framework is well-implemented software that helps me a lot when developing web applications in Rust (that said, HTTP APIs of [a database](https://www.scopedb.io)).

Also there is a group of contributors that are maintaining this crate, as well as quite a few users depending on poem.

I wonder if you guys consider it is a good idea to apply for joining the ASF Incubator [1][2], which, in my experience, could help grow the community in a battle-tested way ([The Apache Way](https://www.apache.org/theapacheway/)).

[1] https://incubator.apache.org/
[2] https://incubator.apache.org/cookbook/

[Apache OpenDAL](https://github.com/apache/opendal) is a good example that shows how a Rust project can grow with the help of ASF Incubator.

I hereby open the discussion to collect feedbacks and I'm glad to answer any questions you concerned.

To clarify, I'm one of ASF Incubator mentors and I can help foster the way to ASF Incubator, including acting as a champion to drive an incubator proposal [3] and resolving the initial membership/staff.

[3] https://incubator.apache.org/guides/proposal.html

**Comment by PsiACE** [comment]
I pushed for the use of poem when I was at databend, back in late '21, when poem was supposed to be one of the best web frameworks in the Rust community, and I think it still is.

I've been involved in the incubation of Apache OpenDAL in the past, and I've seen the community emerge as more contributors and create some better solutions like go binding.

While joining the foundation is not a necessity, it does provide some convenience in a way. I think I'll be happy to see poem there.
